site/components/src/main/java/com/visitscotland/brxm/utils/SiteProperties.java

```diff
@@ -166,4 +166,9 @@ public String getDownloadExtensions() {
         return readString(DOWNLOAD_EXTENSIONS);
     }
 
+    public Integer getEventsListingsPageSize() {
```

Comment:
Is there a reason of why we are using the reference type for Integer over primitive? 

```suggestion
    public int getEventsListingsPageSize() {
```